### PR TITLE
feat:  protoc-gen-go-errors add comment

### PR DESCRIPTION
**Pull request by JeffreyBool**
protoc-gen-go-errors add comment

### CODE REVIEW

**Review by tonybase — APPROVED** [review]

**Comment by JeffreyBool** [comment]
咋样，还有啥问题吗

**Comment by haiyux** [comment]
lint 好像又问题 如果是mac/linux 可以执行一下 make fix 修一下

File: cmd/protoc-gen-go-errors/errors.go [hunk 1]
```diff
@@ -82,21 +82,43 @@ func genErrorsReason(gen *protogen.Plugin, file *protogen.File, g *protogen.Gene
...
 	if len(ew.Errors) == 0 {
 		return true
 	}
 	g.P(ew.execute())
+
 	return false
 }
 
+// GetComment returns content with prefix //
+func GetComment(camelValue, comment string) string {
```
Comment (Cluas) [inline cmd/protoc-gen-go-errors/errors.go:113]: GetComment -> buildComment

**Review by Cluas — APPROVED** [review]
LGTM. Thanks!

**Review by haiyux — APPROVED** [review]

File: cmd/protoc-gen-go-errors/errors.go [hunk 2]
```diff
@@ -82,21 +82,43 @@ func genErrorsReason(gen *protogen.Plugin, file *protogen.File, g *protogen.Gene
...
 }
 
+// buildComment returns comment content with prefix //
+func buildComment(camelValue, comment string) string {
+	if comment == "" {
+		return ""
+	}
+
+	comment = strings.Replace(comment, "//", "", 1)
+	return fmt.Sprintf("// %s %s", camelValue, comment)
```
Comment (Cluas) [inline cmd/protoc-gen-go-errors/errors.go:119]: I think this is an implicit behavior, the user should be responsible for his own Comment, not we generate a camelValue for it

**Review by Cluas — CHANGES_REQUESTED** [review]

**Review by Cluas — APPROVED** [review]

**Review by haiyux — APPROVED** [review]

**Pull request merged**